src/DashboardHub/Bundle/AppBundle/DataFixtures/ORM/LoadDashboard.php

```diff
@@ -44,4 +47,71 @@ public function addDefaultDashboard(ObjectManager $manager)
 
         return $this;
     }
+
+    /**
+     * @param ObjectManager $manager
+     * @param User $user
+     *
+     * @return LoadDashboardData
+     */
+    public function addOldDashboard(ObjectManager $manager, User $user)
+    {
+        $dashboard = new Dashboard();
+        $dashboard->setUid(uniqid('', true));
+        $dashboard->setUser($user);
+        $dashboard->setName('Old Dashboard');
+        $dashboard->setRepository('symfony/symfony');
+        $dashboard->setTheme('DashboardHubAppBundle:Template:GithubTravis.html.twig');
+        $dashboard->setPublic(true);
+        $dashboard->setCreatedOn(new \DateTime('2010-01-01 00:00:00'));
+        $dashboard->setUpdatedOn(new \DateTime());
+        $manager->persist($dashboard);
+
+        return $this;
+    }
+
+    /**
+     * @param ObjectManager $manager
+     * @param User $user
+     *
+     * @return LoadDashboardData
+     */
+    public function addPrivateDashboard(ObjectManager $manager, User $user)
+    {
+        $dashboard = new Dashboard();
+        $dashboard->setUid(uniqid('', true));
+        $dashboard->setUser($user);
+        $dashboard->setName('Private Dashboard');
+        $dashboard->setRepository('zendframework/zf2');
+        $dashboard->setTheme('DashboardHubAppBundle:Template:GithubTravis.html.twig');
+        $dashboard->setPublic(false);
+        $dashboard->setCreatedOn(new \DateTime());
+        $dashboard->setUpdatedOn(new \DateTime());
+        $manager->persist($dashboard);
+
+        return $this;
+    }
+
+    /**
+     * @param ObjectManager $manager
+     * @param User $user
+     *
+     * @return LoadDashboardData
+     */
+    public function addMostPopularDashboard(ObjectManager $manager, User $user)
+    {
```

Comment:
this would seem that it would be a method that calculates on the fly based on views, I assume that you mean my default dashboards or similar